app/bundles/LeadBundle/Views/List/details.html.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
```

Comment:
remove this part, now we do not use this detail

Reply:
Since we are using twig templates now, this file need to be removed anyway, right?